packages/hash/backend-utils/src/logger.ts

```diff
@@ -9,11 +9,26 @@ export type LogLevel = typeof LOG_LEVELS[number];
 
 export type LoggerConfig = {
   mode: "dev" | "prod";
-  level: LogLevel;
+  level?: LogLevel;
   serviceName: string;
   metadata?: Record<string, string>;
 };
 
+const tbdIsLogLevel = (level: string): level is LogLevel =>
+  LOG_LEVELS.includes(level as LogLevel);
+
+const getDefaultLoggerLevel = () => {
+  const envLogLevel = process.env.LOG_LEVEL;
+  return envLogLevel && tbdIsLogLevel(envLogLevel) ? envLogLevel : "info";
+};
```

Comment:
Might be easier to use [`envalid`](https://github.com/af/envalid) + `choices` here, but we can do this next time. They will improve typings for `choices` in 7.3.0 – that might be a good time to switch.

Reply:
Happy to change this when we use `envalid`, but did not think it was appropriate to do so as part of this PR.